spec/lib/tasks/flag_rake_spec.rb

```diff
@@ -0,0 +1,57 @@
+require 'spec_helper'
+
+# see http://robots.thoughtbot.com/test-rake-tasks-like-a-boss
+describe 'flag_reservations:flag_overdue' do
+  include_context 'rake'
+
+  before(:all) { FactoryGirl.create(:app_config) }
+
+  it 'flags reservations due yesterday as overdue' do
+    overdue = FactoryGirl.build(:overdue_reservation, overdue: false)
+    not_overdue = FactoryGirl.build(:valid_reservation)
+    expect(!overdue.overdue)
+    expect(!not_overdue.overdue)
+    subject.invoke
+    expect(overdue.overdue)
```

Comment:
Yea, you're missing your assertions here. You want to test `expect(overdue.overdue).to be_truthy`, and likewise for all of your other expectations. These tests all need to be corrected to make sure they're actually testing stuff :stuck_out_tongue:.